Cargo.toml

```diff
@@ -19,6 +19,7 @@ members = [
   "examples/sprites",
   "examples/todolist",
   "examples/traits",
+  "examples/distributing/uniffi-rust-to-python-library",
```

Comment:
Why the subfolder?

Reply:
Somebody told me to create a `distributing` subfolder.